pkg/volume/csi/csi_attacher_test.go

```diff
@@ -356,6 +358,21 @@ func TestAttacherWithCSIDriver(t *testing.T) {
 			plug, _, tmpDir, _ := newTestWatchPlugin(t, fakeClient, true)
 			defer os.RemoveAll(tmpDir)
 
+			// Use buffered channel here to prevent blocking the reactor
+			attachmentWatchActions := make(chan core.Action, 5)
```

Comment:
nit:  dont know if it worth it to use buffered channel since now we just have one attachmentvolume watcher after prepending the reactor.

Reply:
IMHO I don't want to block the following reactors in any case and I think buffered channel does not cost much more than unbuffered ones, I think this is okay.